docs/community-resources.md

```diff
@@ -0,0 +1,26 @@
+---
+id: community-resources
+title: Community Resources
+sidebar_label: Community Resources
+---
+
+Members of the PayID community have created useful resources for using and learning about PayID.
+
+- [PayID Youtube Tutorial by Ken Carabello](https://www.youtube.com/watch?v=gpLJp1lylns)
+
+- The [PayID Validator](https://payidvalidator.com/) is an [open-source project](https://github.com/rswarthout/payid-validator) that validates responses using [JSON schemas](https://docs.payid.org/payid-interfaces) and verifies whether your PayID server follows various [best practices](payid-best-practices), like:
+
+  - [HTTP response headers](payid-headers)
+  - [Cache-Control](payid-best-practices#cache-control)
+  - [CORS accessibility](payid-best-practices#set-cors-cross-origin-resource-sharing-headers)
+  - Response time
+
+  See also: [Python-based PayID validator](https://github.com/RockHoward/python-payid-validator).
```

Comment:
This isn't really a "See also", these tools validate different things.

The PayID Validator validates PayID server responses.

The "Python-based PayID validator" validates URI strings and sees if they are valid PayIDs.